test_libs/pyspec/eth2spec/utils/minimal_ssz.py

```diff
@@ -16,8 +16,8 @@ def __init__(self, **kwargs):
 
         def __eq__(self, other):
             return (
-                self.fields == other.fields and
-                self.serialize() == other.serialize()
+                    self.fields == other.fields and
```

Comment:
Why 8 space indentation here?

Reply:
Double indentation was from my IDE here, not sure, some style thing. I changed it to one line so it's consistent regardless of formatter.